src/QuarkusConfig.ts

```diff
@@ -43,11 +44,15 @@ export namespace QuarkusConfig {
   }
 
   export function setAlwaysShowWelcomePage(value: boolean): void {
-    workspace.getConfiguration(QUARKUS_CONFIG_NAME).update(removeQuarkusConfigName(ALWAYS_SHOW_WELCOME_PAGE), value, true);
+    saveToQuarkusConfig(ALWAYS_SHOW_WELCOME_PAGE, value);
   }
 
   export function setTerminateProcessOnDebugExit(value: string): void {
-    workspace.getConfiguration(QUARKUS_CONFIG_NAME).update(removeQuarkusConfigName(DEBUG_TERMINATE_ON_EXIT), value, true);
+    saveToQuarkusConfig(DEBUG_TERMINATE_ON_EXIT, value);
+  }
+
+  export function saveToQuarkusConfig<T>(configName: string, value: T) {
+    workspace.getConfiguration(QUARKUS_CONFIG_NAME).update(removeQuarkusConfigName(configName), value, true);
```

Comment:
Could be not replaced with 

```ts
workspace.getConfiguration().update(configName, value, ConfigurationTarget.Workspace);
```

Reply:
I'm not sure if we want to do `ConfigurationTarget.Workspace` here, because that would mean the `alwaysShowWelcomePage` would depend on Workspace settings when I think it should be User/Global settings instead.

I think 'quarkus.tools.debug.terminateProcessOnExit' can be Workspace but I think 'quarkus.tools.starter.api' and 'quarkus.tools.alwaysShowWelcomePage' should be Global/User. @fbricon what do you think?

Reply:
@xorye keep as it worked. Change the ConfigurationTarget to have the same code. But IMHO I think it's more clean to use workspace.getConfiguration().update.

Reply:
Ok great. I agree, `workspace.getConfiguration().update` is much cleaner